system/gameModes/35e/miscinfo.lst

```diff
@@ -27,8 +27,6 @@ BONUSSTACKS:Defense.Dodge.Circumstance.NotRanged.NotFlatFooted
 #
 #misc. currency specific items
 #
-CURRENCYUNIT:Gold
-CURRENCYUNITABBREV:gp
```

Comment:
Abbrev is used in real code

Reply:
ah, I see my problem. I intended to only remove `CURRENCYUNIT`, not `CURRENCYUNITABBREV`. Will fix.